v2/googlecloud-to-elasticsearch/src/main/java/com/google/cloud/teleport/v2/elasticsearch/templates/GCSToElasticsearch.java

```diff
@@ -141,43 +170,68 @@ private static PipelineResult run(GCSToElasticsearchOptions options) {
             || (options.getMaxRetryAttempts() != null && options.getMaxRetryDuration() != null),
         "To specify retry configuration both max attempts and max duration must be set.");
 
+    // Throw error if both Javascript UDF and Python UDF are set. We can only apply one or the
+    // other.
+    checkArgument(
+        !(options.getJavascriptTextTransformGcsPath() != null
+            && options.getPythonExternalTextTransformGcsPath() != null),
+        "Either javascript or Python gcs path must be provided, but not both.");
```

Comment:
nit: Worried just null-checking could be problematic for some automation tools

```suggestion
boolean useJavascriptUdf = !Strings.isNullOrEmpty(options.getJavascriptTextTransformGcsPath());
boolean usePythonUdf = !Strings.isNullOrEmpty(options.getPythonExternalTextTransformGcsPath());
if (useJavascriptUdf && usePythonUdf) {
  throw new IllegalArgumentException(
      "Either javascript or Python gcs path must be provided, but not both.");
}
```
I also think these checks should be put in a common function so they don't have to keep being duplicated in every template